cmd/config/cnv-metrics.yml

```diff
@@ -0,0 +1,124 @@
+  metricName: virt-infra-pod-total-mem-usage-request-delta
+  instant: true
+
+- query: sum(kubevirt_vmi_launcher_memory_overhead_bytes{}) / count (kubevirt_vmi_launcher_memory_overhead_bytes{})
+  metricName: avg-virt-launcher-mem-overhead
+  instant: true
+
+## CPU and Memory related metircs
+- query: avg(avg_over_time(rate(container_cpu_usage_seconds_total{name!="", namespace="openshift-cnv", container!="POD"}[2m])[{{.elapsed}}:])) by (container)
+  metricName: avg-infra-pod-cpu-usage
+  instant: true
+
+- query: max(max_over_time(rate(container_cpu_usage_seconds_total{name!="", namespace="openshift-cnv", container!="POD"}[2m])[{{.elapsed}}:])) by (container)
+  metricName: max-infra-pod-cpu-usage
+  instant: true
+
+- query: avg(avg_over_time(container_memory_rss{name!="", namespace="openshift-cnv", container!="POD"}[{{.elapsed}}:])) by (container)
+  metricName: avg-infra-pod-memory-usage
+  instant: true
+
+- query: max(max_over_time(container_memory_rss{name!="", namespace="openshift-cnv", container!="POD"}[{{.elapsed}}:])) by (container)
+  metricName: max-infra-pod-memory-usage
+  instant: true
+
+- query: avg(avg_over_time(kubevirt_memory_delta_from_requested_bytes[{{.elapsed}}:])) by (container)
+  metricName: avg-infra-container-request-usage-delta
+  instant: true
+
+- query: max(max_over_time(kubevirt_memory_delta_from_requested_bytes[{{.elapsed}}:])) by (container)
+  metricName: max-infra-container-request-usage-delta
+  instant: true
+
+- query: avg_over_time(label_replace(sum(container_memory_rss{pod=~"virt-launcher.*"}) by (pod), "vmi", "$1", "pod", "virt-launcher-(.*)-[^-]+$")[{{.elapsed}}:]) 
+  metricName: avg-mem-usage-by-vmi
+  instant: true
+
+- query: avg_over_time(label_replace(sum(rate(container_cpu_usage_seconds_total{pod=~"virt-launcher.*"}[2m])) by (pod),"vmi", "$1", "pod", "virt-launcher-(.*)-[^-]+$")[{{.elapsed}}:])
+  metricName: avg-cpu-usage-by-vmi
+  instant: true
+
+## storage related metrics
+- query: avg(avg_over_time(rate(kubevirt_vmi_storage_iops_read_total[60s])[{{.elapsed}}:])) by (name) 
+  metricName: avg-vmi-read-iops
+  instant: true
+
+- query: avg(avg_over_time(rate(kubevirt_vmi_storage_iops_write_total[60s])[{{.elapsed}}:])) by (name)
+  metricName: avg-vmi-write-iops
+  instant: true
+
+- query: avg(avg_over_time(rate(kubevirt_vmi_storage_read_traffic_bytes_total[60s])[{{.elapsed}}:])) by (name)
+  metricName: avg-vmi-storage-read-traffic
+  instant: true
+
+- query: avg(avg_over_time((rate(kubevirt_vmi_storage_write_traffic_bytes_total[60s])[{{.elapsed}}:]))) by (name) 
+  metricName: avg-vmi-storage-write-traffic
+  instant: true
+
+## network related metrics
+- query: avg(avg_over_time(rate(kubevirt_vmi_network_receive_bytes_total[60s])[{{.elapsed}}:])) by (name)
+  metricName: avg-vmi-traffic-received
+  instant: true
+
+- query: avg(avg_over_time(rate(kubevirt_vmi_network_transmit_bytes_total[60s])[{{.elapsed}}:])) by (name) 
+  metricName: avg-vmi-traffic-transmitted
+  instant: true
```

Comment:
network usage metrics are not relevant with the tests we have at the moment

Reply:
I suggest keeping those metrics for now. As charles is working on the UDN tests, we might need it soon.